src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,240 @@
+  </p>
+
+  <p>
+    For new components, we recommend that you reach out to the SKY UX team for a "soft" review of the initial implementation before you write unit tests. This allows you to make requested changes to your code before writing the tests.
+  </p>
+
+  <p>
+    When your tests run, code coverage results are generated and can be located in <stache-code>coverage/&lt;browser version>/index.html</stache-code>. You can launch this straight from disk and view the SKY UX unit test code coverage results in your default web browser.
+  </p>
+
+  <stache-page-anchor>
+    Write visual regression tests
+  </stache-page-anchor>
+
+  <p>
+    During continuous integration builds, SKY UX runs visual regression tests through BrowserStack using <a href="http://www.protractortest.org/">Protractor</a> and <a href="https://github.com/koola/pix-diff">pix-diff</a>. These prerequisites are automatically installed when you run <stache-code>npm run test:visual</stache-code> to create and compare screenshots in the <stache-code>skyux-spa-visual-tests/screenshots-baseline-local/</stache-code> folder.
+  </p>
+
+  <p>
+    To create visual tests for a new component, create a folder for the tests in <stache-code>skyux-spa-visual-tests/src/app</stache-code> and add four files for the visual regression test:
+  </p>
+
+  <ul>
+    <li>
+      <stache-code>{{ '{' }}componentName}-visual.component.html</stache-code>: The template for the component to render and take a screenshot of.
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}-visual.component.ts</stache-code>: The TypeScript code for the component to render and take a screenshot of.
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}.visual-spec.ts</stache-code>: The Protractor code to run the screenshot tests.
+    </li>
+    <li>
+      <stache-code>index.html</stache-code>: The route file that contains the component to render and take a screenshot of.
+    </li>
+  </ul>
+
+  <p>
+    To run visual regression tests locally, you must first run them locally against the master branch and then run the tests against your feature branch. Keep in mind that you need to use the same monitor and resolution on both set of tests. If your visual regression tests fail, you can check the failed tests at <stache-code>skyux2\skyux-spa-visual-tests\screenshots-diff-local\diff</stache-code> to find the cause of the failures.
+  </p>
+
+  <stache-page-anchor>
+    Submit code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Commit and push your changes to the repo.
+    </li>
+    <li>
+      Submit a pull request.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Available NPM scripts
```

Comment:
I think it would be safer to remove this section altogether and just rely on our CLI's commands.

Reply:
✔️